charts/argo-workflows/templates/controller/workflow-controller-deployment-pdb.yaml

```diff
@@ -14,6 +14,7 @@ spec:
   {{- else }}
   minAvailable: 0
   {{- end }}
+  unhealthyPodEvictionPolicy: IfHealthyBudget
```

Comment:
Feels like this should be user-configurable as some users may well prefer `AlwaysAllow`

Reply:
Done ✅ 

I ran `./scripts/helm-docs.sh` but the attribute doesn't show up in the doc. What did I do wrong ?